### PR TITLE
fix handle resolution in some cli commands

### PR DESCRIPTION
**Pull request by whyrusleeping**
probably more of this to come, the ProdHandleResolver type is basically broken now

### CODE REVIEW

File: cmd/gosky/admin.go [hunk 1]
```diff
@@ -78,7 +82,7 @@ var checkUserCmd = &cli.Command{
 
 		adminKey := cctx.String("admin-password")
 		xrpcc.AdminToken = &adminKey
-		xrpcc.Host = id.PDSEndpoint()
+		xrpcc.Host = cctx.String("admin-endpoint")
```
Comment (bnewbold) [inline cmd/gosky/admin.go:85]: makes sense. at some point we should probably have a better name than "admin" but i'm not sure what that should be right now

File: cmd/gosky/main.go [hunk 2]
```diff
@@ -765,7 +765,7 @@ var listLabelsCmd = &cli.Command{
 		since := time.Now().Add(-1 * delta).UnixMilli()
 
 		xrpcc := &xrpc.Client{
-			Host: "https://api.bsky.app",
+			Host: "https://mod.bsky.app",
```
Comment (bnewbold) [inline cmd/gosky/main.go:768]: would be good to have this be a config (env var) as well, but not a blocking issue

File: cmd/gosky/admin.go [hunk 3]
```diff
@@ -623,25 +627,25 @@ var takeDownAccountCmd = &cli.Command{
 
 		for _, did := range cctx.Args().Slice() {
 			if !strings.HasPrefix(did, "did:") {
-				phr := &api.ProdHandleResolver{}
-				resp, err := phr.ResolveHandleToDid(ctx, did)
+				dir := identity.DefaultDirectory()
+				resp, err := dir.LookupHandle(ctx, syntax.Handle(did))
```
Comment (bnewbold) [inline cmd/gosky/admin.go:631]: feeling happy that these re-writes are basically one-to-one

**Review by bnewbold — APPROVED** [review]
happy to get this stuff over to the identity package! happy to help with any tweaks to that package if needed, and doing other similar code changes

File: cmd/gosky/admin.go (hunk 1 above)
Comment (whyrusleeping) [reply cmd/gosky/admin.go:85]: yeah, i always lean towards more verbose when i'm unsure here

File: cmd/gosky/admin.go (hunk 3 above)
Comment (whyrusleeping) [reply cmd/gosky/admin.go:631]: yeah, the only weird bit is i'm not using the parse handle code and am just casting direct to the handle type. Feels like unnecessary overhead to me

**Pull request merged**